### DISCUSSION TITLE
Getting the list item attachment in sharepoint search

### DISCUSSION THREAD

**Discussion by ferrarirosso** [microsoft-search/pnp-modern-search#3071] [General]
Is there a way to get the exact list attachment from the spo list item which caused the search query to be hit
I am aware that there is the LinkOfficeChild property, does a property exists which gives me in which attachment the search query was found ?

**Comment by wobba** [comment]
Sure, and you can add LinkOfficeChild to the list of properties to return. I have an old post on this at https://www.techmikael.com/2014/04/solution-to-displaying-attachments-for.html so using some string splitting with handlebar helpers you should get all the URL's.

**Comment by ferrarirosso** [comment]
Thank you for your answer.
If I understand the blog post correctly, you can display the url of the attachments, but, referring to the example in your blog post, if I am searching for string x and this string is only in the document "score.docx" and not in the document "testlongdoc.docx", how would I know that the hit regards "score.docx" ? Is this possible ?

**Reply by wobba** [reply]
No. All attachements are indexed as one item in the index, together with the list item itself. Think of them as document pages. If you want per attachment you need to store them in a document library.

**Comment by ferrarirosso** [comment]
Would be a nice addon for the future, thx
